lib/rubocop/cop/style/redundant_double_splat_hash_braces.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for redundant uses of double splat hash braces.
+      #
+      # @example
+      #
+      #   # bad
+      #   do_something(**{foo: bar, baz: qux})
+      #
+      #   # good
+      #   do_something(foo: bar, baz: qux)
+      #
+      class RedundantDoubleSplatHashBraces < Base
+        extend AutoCorrector
+
+        MSG = 'Remove the redundant double splat hash braces.'
```

Comment:
"double splat preceding the hash literal" sounds better to me.

Reply:
Actually, braces are redundant as well as double splat.

```ruby
# bad
do_something(**{foo: bar, baz: qux})

# good
do_something(foo: bar, baz: qux)
```

For example, what about the following message?

```
Remove redundant double splat and braces, use keyword arguments directly.
```

Reply:
Fine by me.

Reply:
I've updated it. Thank you!